low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server_manifest.py

```diff
@@ -0,0 +1,345 @@
+        kvs_src = None
+        kvs_dst = None
+        response, err, returncode = SimpleProcess("lshw -json").run()
+        if returncode:
+            msg = f"Failed to capture Node support data. Error:{str(err)}"
+            logger.error(self.log.svc_log(msg))
+            raise ManifestError(errno.EINVAL, msg)
+        try:
+            with open(LSHW_FILE, 'w+') as fp:
+                json.dump(json.loads(response.decode("utf-8")), fp,  indent=4)
+            with open(MANIFEST_OUTPUT_FILE, 'w+') as fp:
+                json.dump({}, fp,  indent=4)
+            kvs_src = KvStoreFactory.get_instance(f'json://{LSHW_FILE}').load()
+            kvs_dst = KvStoreFactory.get_instance(f'json://{MANIFEST_OUTPUT_FILE}')
+        except Exception as e:
+            msg = "Error in getting {0} file: {1}".format(LSHW_FILE, e)
+            logger.error(self.log.svc_log(msg))
+            raise ManifestError(errno.EINVAL, msg)
+        return kvs_src, kvs_dst
+
+    def map_manifest_server_data(self, field, manifest_key, data, kv_key):
+        parent_id = ""
+        base_key = '>'.join(kv_key.split('>')[:-1])
+        if base_key:
+            value = data.get(base_key+'>'+field)
+        else:
+            value = data.get(field)
+        value = value.replace(" (To be filled by O.E.M.)", "") \
+            if value else 'NA'
+        if field == 'id' and '>' in kv_key:
+            parent_key = '>'.join(kv_key.split('>')[:-2])
+            field = '>'+field if parent_key else field
+            parent_id = data.get(parent_key+field)+"-"
+        self.kv_dict[manifest_key] = parent_id + value
+    
+    def get_manifest_output_data(self):
+        data = {}
+        try:
+            with open(MANIFEST_OUTPUT_FILE) as json_file:
+                data = json.loads(json_file.read())
+        except Exception as e:
+            msg = "Error in getting {0} file: {1}".format(json_file, e)
+            logger.error(self.log.svc_log(msg))
+            raise ManifestError(errno.EINVAL, msg)
+        try:
+            if os.path.exists(LSHW_FILE):
+                os.remove(LSHW_FILE)
+            if os.path.exists(MANIFEST_OUTPUT_FILE):
+                os.remove(MANIFEST_OUTPUT_FILE)
+        except OSError as ex:
+            msg = f"Failed in manifest tmp files cleanup. Error:{str(ex)}"
+            logger.warn(self.log.svc_log(msg))
+        return data
+
+    def get_cortx_service_info(self):
+        """Get cortx service info in required format."""
+        cortx_services = self.service.get_cortx_service_list()
+        cortx_service_info = self.get_service_into(cortx_services)
+        return cortx_service_info
+
+    def get_external_service_info(self):
+        """Get external service info in required format."""
+        external_services = self.service.get_external_service_list()
+        external_service_info = self.get_service_into(external_services)
+        return external_service_info
+
+    def get_service_into(self, services):
```

Comment:
Please correct the typo `get_service_into` -> `get_service_info`

Reply:
Done